controls/radspreadsheet/events.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Events
+page_title: Events
+description: Events
+slug: radspreadsheet-events
+tags: events
+published: True
+position: 8
+---
+
+# Events
+
+This topic describes the events you can use in RadSpreadsheet so you can get a notification about different actions.
+
+* [RadSpreadsheet Events](#radspreadsheet-events)
+* [Workbook Events](#workbook-events)
+* [Worksheet Events](#worksheet-events)
+* [RadWorksheetEditor Events](#radworksheeteditor-events)
+* [Cells Events](#cells-events)
+* [Row/Column Events](#rowcolumn-events)
+
+
+## RadSpreadsheet Events
+
+* **ActiveSheetChanged**: Occurs when the active sheet is changed.
+
+* **ActiveSheetEditorChanged**: Occurs when the active sheet editor is changed.
+
+* **MessageShowing**: Occurs when the user is presented with a warning message. The event arguments are of type **MessageShowingEventArgs** and expose the following properties:
+	* **IsHandled**: Gets or sets whether the event is handled.
+	* **Header**: Gets or sets the header of the message.
+	* **Content**: Gets or sets the content of the message.
+	* **NotificationType**: Gets or sets the type of the notification. The property is of type [MessageBoxNotificationType](http://docs.telerik.com/devtools/wpf/api/html/t_telerik_windows_controls_spreadsheet_dialogs_messageboxnotificationtype.htm).
+
+	The code from **Example 1** shows how to disable the messages appearing when the user is trying to edit a protected worksheet.
+
+	#### **[C#] Example 1: Disable messages related to protection using the MessageShowing event**
+	{{region radspreadsheet-events_0}}
+
+		public void AttachToMessageShowingEvent()
+		{
+		    this.radSpreadsheet.MessageShowing += radSpreadsheet_MessageShowing;
+		}
+		
+		private void radSpreadsheet_MessageShowing(object sender, Telerik.Windows.Controls.Spreadsheet.MessageShowingEventArgs e)
+		{
+		    if (e.NotificationType == Telerik.Windows.Controls.Spreadsheet.Dialogs.MessageBoxNotificationType.ProtectedWorksheetError)
+		    {
+		        e.IsHandled = true;
+		    }
+		}
+	{{endregion}} 
+
+* **WorkbookCommandError**: Notifies that an error occurred while executing a command in RadSpreadsheet. The arguments are of type **CommandErrorEventArgs** and you can use the Exception property to check what exactly the error is.
+
+* **WorkbookChanging**: Occurs when the workbook is changing.
```

Comment:
Here we can use the same wording as with the WorkbookCommandExecuting. Example: "Occurs when the workbook starts changing".